### DISCUSSION TITLE
Allow multiple URL parameters with same name

### DISCUSSION THREAD

**Discussion by kgroshert** [grafana/grafana-infinity-datasource#293] [Ideas]
I have a web-api of a commercial product (so I unfortunately cannot change it) which needs multiple identical parameters passed in the URL like:

https://......./show.py?state=ok&column=hostname&column=uptime&column=ipaddress

If i do this inside the URL in Infinity it works of course but if I add the params to the "Query params" only one of them makes it through.

It looks fine in the Dashboard-JSON but I suspect it gets overwritten when creating the URL later?

        "params": [
          {
            "key": "column",
            "value": "hostname"
          },
          {
            "key": "column",
            "value": "uptime"
          },
          {
            "key": "column",
            "value": "ipaddress"
          }
        ]

Is there anything that could be done?

**Comment by yesoreyeram** [comment]
Can you try passing those in the URL field instead params? May work

**Reply by kgroshert** [reply]
Yes if I put it in the URL it works. Would be nice to have it as an query param though as the URL is already pretty long and it's hard to edit the URL in the single-line URL field. But it certainly works as a workaround!

**Reply by yesoreyeram** [reply]
Sure.. good point. Fix coming in next release
